### PR TITLE
Add PayboxResponse tests

### PR DESCRIPTION
**Pull request by soullivaneuh**
About exception test: https://thephp.cc/news/2016/02/questioning-phpunit-best-practices

### CODE REVIEW

File: tests/Response/PayboxResponseTest.php [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    /**
+     * @var PayboxResponse
+     */
+    private $response;
+
+    protected function setUp()
+    {
+        $this->response = new PayboxResponse([
+            'CODEREPONSE' => 1337,
+            'COMMENTAIRE' => 'French keywords is bad, and you should feel bad!',
```
Comment (greg0ire) [inline tests/Response/PayboxResponseTest.php]: "are bad"

File: tests/Response/PayboxResponseTest.php [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    }
+
+    public function testNotExistsProperty()
+    {
+        $this->assertNull($this->response->getDiscount());
+    }
+
+    /**
+     * @expectedException \BadMethodCallException
+     * @expectedExceptionMessage Undefined method getFake
```
Comment (greg0ire) [inline tests/Response/PayboxResponseTest.php]: You could use `setExceptedException()` instead :)

Comment (soullivaneuh) [inline tests/Response/PayboxResponseTest.php]: Yes I can.

...

But I prefer this way. :heart:

Comment (greg0ire) [inline tests/Response/PayboxResponseTest.php]: Annotations are bad and you should feel bad :trollface:

**Pull request merged**